node_modules/oae-core/aboutcontent/aboutcontent.html

```diff
@@ -53,7 +53,11 @@ <h3 id="aboutcontent-title">${contentProfile.displayName|encodeForHTML}</h3>
             </div>
             {if contentProfile.resourceSubType === 'link'}
                 <div id="aboutcontent-link">
-                    <a href="${contentProfile.link}" title="${contentProfile.displayName|encodeForHTMLAttribute}" target="_blank">${contentProfile.link|encodeForHTML}</a>
```

Comment:
It might be better to do:

```
<a href="${contentProfile.link}" {if contentProfile.displayName !== contentProfile.link} title="${contentProfile.displayName|encodeForHTMLAttribute}" {/if} target="_blank">${contentProfile.link|encodeForHTML}</a>
```

So we don't have to repeat the markup of the link